fluster/decoders/gstreamer.py

```diff
@@ -257,7 +257,8 @@ class GStreamerLibavVP8(GStreamer10Video):
     """GStreamer VP8 Libav decoder implementation for GStreamer 1.0"""
 
     codec = Codec.VP8
-    decoder_bin = " ivfparse ! avdec_vp8 "
+    check_decoder_bin = " ivfparse ! avdec_vp8 "
```

Comment:
IIRC check_decoder_bin doesn't need the parser.
```suggestion
    check_decoder_bin = " avdec_vp8 "
```

Reply:
@rgonzalezfluendo does this apply to all the decoder entries i assume?

Reply:
Right.

Reply:
@benjamingaignard sorry for the inconveniences. Could you apply this change to all the decoders you modified Thanks for your support.